db/migrate/20150514114044_migrate_ptables_to_templates.rb

```diff
@@ -35,16 +35,14 @@ def up
       new_ptable.os_family = old_ptable.os_family
       new_ptable.type = 'Ptable'
       new_ptable.operatingsystems = old_ptable.operatingsystems
-      new_ptable.hostgroups = old_ptable.hostgroups
       new_ptable.save!
 
       Audit.where(:auditable_type => 'Ptable', :auditable_id => old_ptable.id).each do |audit|
         audit.update_attribute :auditable_id, new_ptable.id
```

Comment:
While we're touching this migration, we can also get rid of this loop using `update_all`.